frontend/src/concepts/modelRegistry/types.ts

```diff
@@ -92,19 +97,35 @@ export type ModelRegistryBase = {
   customProperties: ModelRegistryCustomProperties;
 };
 
-export type ModelArtifact = ModelRegistryBase & {
-  uri?: string;
-  state?: ModelArtifactState;
-  author?: string;
-  modelFormatName?: string;
-  storageKey?: string;
-  storagePath?: string;
-  modelFormatVersion?: string;
-  serviceAccountName?: string;
-  modelLocationType?: ModelLocationType;
-  artifactType: string;
+export type PipelineRunReference = {
+  project: string;
+  runId: string;
+  runName: string;
 };
 
+export type ModelSourceProperties = {
+  modelSourceKind?: ModelSourceKind;
+  modelSourceClass?: string;
+  modelSourceGroup?: string;
+  modelSourceName?: string;
+  modelSourceId?: string;
+  pipelineRunReference?: PipelineRunReference;
```

Comment:
@YuliaKrimerman this isn't a property in the MR API, PipelineRunReference was just for convenience in passing around project/runId/runName where you already were. We need to remove it here. It doesn't appear to be used anywhere anyway